tethys_apps/cli/__init__.py

```diff
@@ -82,7 +82,7 @@ def tethys_command():
     gen_parser.add_argument('--overwrite', dest='overwrite', action='store_true',
                             help='Overwrite existing file without prompting.')
     gen_parser.set_defaults(func=generate_command, allowed_host=None, allowed_hosts=None, client_max_body_size='75M',
-                            uwsgi_processes=10, db_username='tethys_default', db_password='pass', db_port=5436,
+                            asgi_processes=4, db_username='tethys_default', db_password='pass', db_port=5436,
```

Comment:
Since ASGI is asynchronous all you really need is one. the number of connections Daphne can handle will depend on the machine. Documentation recommends to set default to four since four cores is normal nowadays. If you want to increase it then they recommend one extra daphne instance per core. In short, 10 process would be okay only if the machine has 10 cores.

Reply:
I fixed the help to "defaults to 4"